src/GitHub/Data/Repos.hs

```diff
@@ -61,12 +61,23 @@ data Repo = Repo
     , repoPushedAt        :: !(Maybe UTCTime)   -- ^ this is Nothing for new repositories
     , repoCreatedAt       :: !(Maybe UTCTime)
     , repoUpdatedAt       :: !(Maybe UTCTime)
+    , repoPermissions     :: !(Maybe RepoPermissions)
```

Comment:
It is not clear to me in what sense permissions are a property of a repository.  I would think that permissions is how a user relates to a repository.

So this would need documentation:
- What would it mean that a repo has no permission (`Nothing`)?
- What would it mean that a repo has `push` permissions?

A link to the original GitHub API docs would also be in order, so that the reader can get to the relevant section by a simple click and read more there.

Reply:
> a property of a repository.

Yeah it's definitely not returning these from the same database table ;)

I've documented that the permissions relate to the authenticated user.



> a simple click and read more there.

If only there was more to read. GitHub doesn't document their response object fields. I have updated the links so readers can visit the void themselves.

Reply:
Thanks!